app/src/bin/crate.in.sh

```diff
@@ -67,17 +67,23 @@ ERROR: Garbage collection log directory '$GC_LOG_DIR' does not exist or is not a
 EOF
     exit 1
   fi
+  LOGGC="$GC_LOG_DIR/gc.log"
 
-  JAVA_OPTS="$JAVA_OPTS -Xloggc:$GC_LOG_DIR/gc.log"
-  JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCDetails"
-  JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCDateStamps"
-  JAVA_OPTS="$JAVA_OPTS -XX:+PrintTenuringDistribution"
-  JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCApplicationStoppedTime"
-  # GC logging requires 16x64mb = 1g of free disk space
-  JAVA_OPTS="$JAVA_OPTS -XX:+UseGCLogFileRotation"
-  JAVA_OPTS="$JAVA_OPTS -XX:NumberOfGCLogFiles=$GC_LOG_FILES"
-  JAVA_OPTS="$JAVA_OPTS -XX:GCLogFileSize=$GC_LOG_SIZE"
-
+  JAVA_VERSION=`java -version 2>&1 | grep "version" | awk '{print $3}' | tr -d \" | awk '{split($0, array, ".")} END{print array[2]}'`
```

Comment:
Minor: Could avoid the extra grep call by changing the awk call: `awk '/version/ {print $3}'`
Would probably be also possible by doing all at once with awk, by using `gsub`, though it might be cleaner as it is

Reply:
also, `tr -d \"` is not necessary